unified-codes/apps/data-loader/src/app/v2/DataParser.ts

```diff
@@ -609,7 +609,7 @@ export class DataParser {
         // });
 
         // Process external properties at product (UC2) level
-        if (!uc7 && uc2) {
+        if (uc2) {
```

Comment:
I couldn't see this causing any issues (but happy to be told I'm wrong 😆 ), needed for RxNav and NZULM codes to show up for the COVID vaccine products as they had a uc7 but still needed properties assigned at the uc2 level

Reply:
IIRC there was an issue because UC7 wasn't implemented in our code.
Importing the UC2 value for items that had UC7 resulted in something unwanted - I would think parents with no children? Should be ok now though